processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -186,7 +186,8 @@ public AlphaNumericTopNMetricSpec(
       @JsonProperty("previousStop") String previousStop
   )
   {
-    super(previousStop);
+    // Set empty string as previousStop of super class otherwise we will get NPE
+    super((previousStop == null) ? "" : previousStop);
     this.previousStop = (previousStop == null) ? "" : previousStop;
```

Comment:
looks like we are having two variables for previousStop in parent class and child class, 
can we also move them into a single one ?